build/docker/run.sh

```diff
@@ -25,6 +25,8 @@ DOCKER_TAG=$DOCKER_REPO:$DISTRO
 
 printenv | sort
 
+composer install
```

Comment:
This looks like it should be part of the php section of the Dockerfile.  By putting it in the run section, we run "composer install" on the host and then start a docker environment which doesn't have access to it.